src/Model/Package.php

```diff
@@ -34,7 +34,34 @@ public function getTitle()
      */
     public function getSummary()
     {
-        return $this->renderWith('Package_summary');
+        $summary = $this->renderWith('Package_summary');
+        $this->extend('updateSummary', $summary);
+        return $summary;
+    }
+
+    /**
+     * Gives the summary template {@see getSummary()} a list of badges to show against a package
+     *
+     * badgeDefinitions are in the format [$title => $type] where:
+     *   title is the unique string to display
+     *   type is an optional class attribute (applied as a BEM modified, by default)
+     *
+     * @return ArrayList
+     */
+    public function getBadges()
+    {
+        $badgeDefinitions = [];
+        $this->extend('updageBadges', $badgeDefinitions);
```

Comment:
Typo* also might be good to put the extension point after the logic below that assembles the list

Reply:
lol, somewhere between 'update' and 'upgrade', ahaha